schemas/2018-07-01/Microsoft.Storage.json

```diff
@@ -105,6 +115,64 @@
       ],
       "description": "Microsoft.Storage/storageAccounts"
     },
+    "storageAccounts_blobServices": {
```

Comment:
>storageAccounts_blobServices [](start = 5, length = 28)

do you need to add this to 2015-01-01/deploymentTemplate.json and 2014-04-01-preview/deploymentTemplate?

Reply:
Blob services APIs are supported only in 2018-07-01 and later SRP API versions, so I think we do not want to add it to those versions here?

I actually am not sure why all the existing objects have 2015-01-01 in 'oneOf', on the side of the type ref. I'm new to Azure schema. Any comments/suggestions?

Reply:
Tiano: I've added BlobServices in those deployment template files in latest commit. Kindly take a look.